fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/ViewHelpers.java

```diff
@@ -212,7 +212,10 @@ public String getObjectTitle(final Graph graph, final Node subject) {
             final Iterator<Triple> objects = getObjects(graph, subject, p);
```

Comment:
here's a ticket for cleaning up the class: https://jira.duraspace.org/browse/FCREPO-1813